### ISSUE TITLE
Determine if we can remove the dcterms and eperson metadata schemas from the metadata registry.

### ISSUE THREAD

**Issue by alawvt** [VTUL/vtechworks#97]
We need only metadata schemas: 
dc, http://dublincore.org/documents/dcmi-terms/, and thesis, http://www.ndltd.org/standards/metadata/etdms/1.0/, in the metadata registry, http://vtechworks.lib.vt.edu/admin/metadata-registry. 

We'd like to remove the eperson, http://dspace.org/eperson, and dcterms, http://purl.org/dc/terms/,  schemas from the registry to reduce the number of choices in the field drop down list, etc. Since the system wouldn't let @amandafrench delete the dcterms schema, we need to check that no items have been assigned fields from these schemas.
We want to also check if there are other reasons to leave these metadata schemas in the registry.

**Comment by amandafrench** [comment]
Retitling this issue assigning to @keithgee to look into. I feel like dc.terms it has something important to do with Linked Open Data. Still, I'm definitely open to deleting it if we really don't need it.

**Comment by amandafrench** [comment]
Hmm, there's a fair discussion here: https://wiki.duraspace.org/display/DSPACE/Proposal+to+Update+DC+Registry+and+Add+DCTERMS+Registry

**Comment by alawvt** [comment]
@keithgee pointed out that some DCTERMS are in the ```<head>``` of the HTML page for each item. For example, in http://vtechworks.lib.vt.edu/handle/10919/23111,
```
<meta name="DCTERMS.dateAccepted" content="2013-05-30T08:00:14Z" scheme="DCTERMS.W3CDTF" />
<meta name="DCTERMS.available" content="2013-05-30T08:00:14Z" scheme="DCTERMS.W3CDTF" />
<meta name="DCTERMS.issued" content="2013-05-29" xml:lang="en_US" scheme="DCTERMS.W3CDTF" />
<meta name="DC.identifier" content="vt_gsexam:1000" xml:lang="en_US" />
<meta name="DC.identifier" content="http://hdl.handle.net/10919/23111" scheme="DCTERMS.URI" />
<meta name="DCTERMS.abstract" content="The development of a fleet..." xml:lang="en_US" />
<meta name="DCTERMS.medium" content="ETD" xml:lang="en_US" />
```
However, these are not the fields that we (or, rather, the ETD ingest script) added. They were somehow crosswalked from our DC fields. I see another example of this:

```
<meta name="DC.creator" content="Pyrak, Matthew James" xml:lang="en_US" />
```
which is crosswalked from our DC.contributor.author.

We need to ensure that removing the DCTERMS metadata schema won't affect this crosswalk, nor any that might exist for OAI-PMH.

All 4 metadata schemas are in vtechworks.dev and vtechworks.pprd, so I can't compare it to those.

**Comment by amandafrench** [comment]
I am becoming increasingly convinced that deleting these registries will cause widespread unanticipated technical difficulties in the future, including potentially making it harder for other systems to index and aggregate our content, for what is after all a small local benefit to 3 or 4 people: having fewer choices to scan through when adding items or editing metadata. The more I read about it in the Duraspace discussion I referenced above, the more foolish it seems to delete a registry which 1) is semantic, 2) is a more recent schema recommended by the Dublin Core working group, 3) is apparently going to be the DSpace default, 4) went through extensive international peer discussion by experts in the field. 

I would suggest instead that we keep the registries and delete individual fields, which is perfectly feasible (I deleted dc.subject.ddc for Dewey Decimal Classification Number, no problem, on dev). Have added this to the fairly major project in the backlog of overhauling the registries and writing usage guidelines. Might be a sole candidate for January sprint (animal starting with E?)
